en/protocol/landing_target.md

```diff
@@ -0,0 +1,71 @@
+# Landing Target Protocol
+
+The landing target protocol/message communicates the position of one or more targets from MAVLink positioning system(s) to an autopilot.
+A multicopter or VTOL system can use the message to land with far greater positional accuracy than provided by conventional GPS
+(GPS provides position within several meters while a landing-target system might reasonably provide centimetre-level precision landing).
+
+Positioning system might typically consist of an onboard companion computer with a vision system that can detect a light beacon or target image.
+Radio beacons and different types of visual markers and tags are also supported.
+
+
+## Protocol Messages
+
+The message used by this protocol is [LANDING_TARGET](../messages/common.md#LANDING_TARGET).
+This is *broadcast* by the positioning system to indicate the position of a particular target at a particular time.
+A typical broadcast rate would be (TBD).
```

Comment:
@ndepal Any idea on a minimal/typical "useful" broadcast rate for a positioning message?

Reply:
With the IRLock camera, you get 40 - 50 Hz (but that doesn't come in via MAVLink from offboard). But I guess you could work with ~10Hz depending on desired accuracy, landing speed, etc. I believe @TSC21 has used precision landing with offboard target tracking, so maybe he can provide some more input.

Reply:
OK. Well for now I'll make this "general guidance":

> The required broadcast rate depends on the landing speed and desired accuracy; start with rates between 10 Hz and 50 Hz and tune performance as needed.